src/nodeDebugAdapter.ts

```diff
@@ -193,7 +194,7 @@ export class NodeDebugAdapter extends ChromeDebugAdapter {
 
         this._captureFromStd = args.outputCapture === 'std';
 
-        const resolvedProgramPath = await this.resolveProgramPath(programPath, args.sourceMaps);
```

Comment:
Handle a missing `__debuggablePatterns`, just to be safe when there's a node/node2 version mismatch or something.

Reply:
Sounds good. Wasn't sure how tightly those are bound